docs/001_develop/03_client-capabilities/005_grids/001_grid-pro/01_grid-pro.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Grid Pro'
+sidebar_label: 'Overview'
+id: grid-pro
+keywords: [genesis, foundation, ui, grid, grid-pro, ag]
+tags:
+  - connected
+  - datasource
+  - grid
+  - grid-pro
+  - ag
+---
+
+# Grid Pro
+
```

Comment:
We need more of an intro here. Why use Grid Pro over tabulator etc?

We have lots of details https://learn.genesis.global/docs/web/web-components/grids/grid-pro/grid-pro-intro/ I'm sure we could extract from

Reply:
see https://github.com/genesiscommunitysuccess/docs/pull/1857#discussion_r1827852286